styles/scss/core/_typography.scss

```diff
@@ -1,11 +1,23 @@
 @import "~bootstrap/scss/type";
 
 @mixin mobile-type {
-  .display-1,
-  .display-2,
-  .display-3,
+  .display-1 {
+    font-size: $display1-mobile-size;
+    line-height: $display-mobile-line-height;
+  }
+
+  .display-2 {
+    font-size: $display2-mobile-size;
+    line-height: $display-mobile-line-height;
+  }
+
+  .display-3 {
+    font-size: $display3-mobile-size;
+    line-height: $display-mobile-line-height;
+  }
+
   .display-4 {
-    font-size: $display-mobile-size;
+    font-size: $display4-mobile-size;
     line-height: $display-mobile-line-height;
   }
```

Comment:
Looking at this it seems we used to use the same size and line height for `.display-1` through `.display-4`, and this updates it so we have different sizes but not different line heights? Just curious as to the reasoning behind that.

Reply:
Great question! So, the new 2U Paragon Elm Theme has distinct sizes for mobile `.display-1` through `.display-4`, but Paragon `alpha` doesn't currently actually have the tokens/CSS support for level-specific mobile display font sizes. As such, this PR introduces tokens for distinct levels of display size for mobile. 

Regarding why there's not currently level-specific mobile line heights is that the Elm Theme continues to rely on a single line height across all mobile display levels (i.e., `line-height: 1`), which also happens to be the non-mobile default line height as well.

Paragon base theme defines `display-mobile-line-height: 3.5rem`, and Elm Theme overrides this to be `display-mobile-line-height: 1`.

There is also only a single line height value configured for all non-mobile display levels, too.

Note that using `line-height: 1` (i.e., without a unit like `rem`) defines the line height to be equal to the font-size which is distinct per level. Given this, `line-height: 1` is technically still results in distinct _computed_ line heights per mobile display level.

Reply:
That sounds very reasonable! I do wonder if _supporting_ different custom line heights for each is something we want to do. I'd lean towards thinking we _should_, but I could definitely be convinced otherwise.

Reply:
Yeah, supporting that as a use case is how I'm leaning, FWIW. Though, maybe we roll with it as is for this PR and @PKulkoRaccoonGang's related PR(s) to do the style-dictionary upgrade to v4, and treat level-specific display line heights as a standalone fast follow to these PRs before `master` -> `alpha`, given it's not necessarily related to the v4 upgrade itself per se and would probably be a reasonably scoped individual contribution.